jupyter/radon/radon.ipynb

```diff
@@ -0,0 +1,1961 @@
+    "[cty.dat](http://www.stat.columbia.edu/~gelman/arm/examples/radon/cty.dat)\n",
+    "respectively.\n",
+    "For this notebook, we have downloaded and renamed them:\n",
+    "\n",
+    "- [data/raw_radon.csv](data/raw_radon.csv) - home radon measurements, and the floor on which the measurement was taken,  \"0\" for basement,  \"1\" for ground floor.\n",
+    "\n",
+    "- [data/raw_uranium.csv](data/raw_uranium.csv) -county level measurements of soil uranium levels in parts per million.\n",
+    "\n",
+    "There are a total of 120K home radon measurements from 3000 US counties.\n",
+    "The per-county measurements follow the population density; there are few or no measurements\n",
+    "for sparsely populated counties, i.e. rural counties and correspondingly more for metropolitan counties.\n",
+    "\n",
+    "**Processed data**\n",
+    "\n",
+    "Our analysis will use both the home radon data measurements and the county level data.\n",
+    "\n",
+    "We need to extract and combine that subset of the information in these tables into the dataset required for this analysis.\n",
+    "The essential pre-processing steps are\n",
+    "\n",
+    "1. Cross-index the county-level data and the home-level data.\n",
+    "\n",
+    "2. Put home radon and soil uranium on the log scale, following Gelman and Hill, chapter 4, section 12.\n",
+    "\n",
+    "3. Restrict the dataset to Minnesota.\n",
+    "\n",
+    "See [Appendix B](#data-prep) for the full set of pre-processing operations.\n",
+    "\n",
+    "The results are in two files\n",
+    "\n",
+    "+ [data/mn_radon.csv](data/mn_radon.csv) contains the individual home radon measurements.\n",
+    "+ [data/mn_counties.csv](data/mn_counties.csv) contains county-level data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a7798292",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mn_radon = pd.read_csv(os.path.join('data','mn_radon.csv'))\n",
+    "mn_radon[:7]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a6ea3e57",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mn_counties = pd.read_csv(os.path.join('data','mn_uranium.csv'))\n",
+    "mn_counties[:3]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4497f7fd",
+   "metadata": {},
+   "source": [
+    "## Best Practice #1: preliminary data analysis\n",
+    "\n",
+    "A sometimes overlooked issue when doing model criticism and model comparison is the fact that we are evaluating the fit of the model _to the data_.  The size and shape of the data informs our choice of model.  Finally, the data collected is not always the data expected.  Therefore we start with plots and summaries of the raw data. When plotting radon measurements against other data elements, we always plot `log_radon` on the y axis, since it is the outcome variable of interest for this case study.\n",
+    "\n",
+    "**A brief introduction to plotnine**\n",
```

Comment:
perhaps it deserves its own appendix?

Reply:
I think it needs _some_ treatment in the body of the case study since it is necessary to at least be able to break down a plotnine command into its parts in order to follow a lot of the rest of the case study

Reply:
rewrote this and moved it up in the order of presentation.